packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -264,6 +272,7 @@ Future<LaunchResult> _startApp(DriveCommand command) async {
       observatoryPort: command.observatoryPort,
     ),
     platformArgs: platformArgs,
+    prebuiltApplication: !command.shouldBuild,
```

Comment:
Rather than negate this here I would expose the negative in the getter as `shouldSkipBuild`

Reply:
I guess you use it the other way too...NBD then